app/api/admin_statistics_api/events.py

```diff
@@ -0,0 +1,55 @@
+from flask_rest_jsonapi import ResourceDetail
+from marshmallow_jsonapi.flask import Schema
+from marshmallow_jsonapi import fields
+from datetime import datetime
+import pytz
+
+from app.api.helpers.utilities import dasherize
+from app.api.bootstrap import api
+from app.models import db
+from app.models.event import Event
+from app.api.data_layers.NoModelLayer import NoModelLayer
+
+
+class AdminStatisticsEventSchema(Schema):
+    """
+    Api schema
+    """
+    class Meta:
+        """
+        Meta class
+        """
+        type_ = 'admin-statistics-event'
+        self_view = 'v1.admin_statistics_event_detail'
+        inflect = dasherize
+
+    id = fields.String()
+    draft = fields.Method("events_draft_count")
+    published = fields.Method("events_published_count")
+    past = fields.Method("events_past_count")
+    total = fields.Method("events_total_count")
+
+    def events_draft_count(self, obj):
+        return Event.query.filter_by(state='draft').count()
```

Comment:
Use the `get_count` helper already present in the project. It is faster than this. (https://gist.github.com/hest/8798884)